zip-0226.rst

```diff
@@ -281,11 +289,17 @@ For every Custom Asset that is burnt, we add to the :math:`\mathsf{assetBurn}` v
 
 .. math:: \mathsf{assetBurn} = \{ (\mathsf{AssetBase},\mathsf{v^{AssetBase}})\ |\ \forall\ \mathsf{AssetBase}\ \textit{s.t.}\ \mathsf{v^{AssetBase}} \neq 0 \}
 
+We denote by :math:`\mathsf{L}` the cardinality of the :math:`\mathsf{assetBurn}` set.
```

Comment:
```suggestion
We denote by :math:`L` the cardinality of the :math:`\mathsf{assetBurn}` set.
```
Using normal math for set sizes, since it will be in the sum indices.